### PR TITLE
Site updated

### PR DESCRIPTION
**Pull request by gc824**
1. Update menu style sheet
1. Updated CSS added header, topnav
1. Updated table for the menu page
1. In the menu page included additional line items

### CODE REVIEW

**Review by rbaia28 — DISMISSED** [review]

File: css/style.css [hunk 1]
```diff
@@ -1,3 +1,5 @@
 h1 {}
 
-<!::-webkit-search-decoration>
\ No newline at end of file
+
+
+
```
Comment (billythakidd04) [inline css/style.css]: These empty lines are useless. They should be deleted.

File: css/style.css [hunk 2]
```diff
@@ -1,3 +1,5 @@
 h1 {}
```
Comment (billythakidd04) [inline css/style.css]: This is doing nothing at the moment. Is that intentional?

File: index.html [hunk 3]
```diff
@@ -9,22 +9,17 @@
...
+    <img src="img/PizzaParty.jpg" alt="Two slices of Pizza holding hands" width="300" height="400">
+    <img src="img/tmp.gif" alt="DJ CAT" width="300" height="400">
 
     <ul>
-        <a href="./index.html">Best Pizza in Town!</a>
-        <li><a href="./ourstory.htm">Our Story</a></li>
-        <li><a href="./Menu.htm"> Menu </a></li>
-        <li><a href="./Reservations.htm">Reservations</a></li>
-        <li><a href="./contactus.htm">Connect with Us!</a></li>
+        <a href=" ./index.html ">Best Pizza in Town!</a>
```
Comment (billythakidd04) [inline index.html]: this link is not correct. It should not be `" ./index.html "`, there shouldn't be spaces and there is no use for the `./`

File: index.html [hunk 4]
```diff
@@ -9,22 +9,17 @@
...
+        <li><a href="./ourstory.htm ">Our Story</a></li>
```
Comment (billythakidd04) [inline index.html]: extra space at the end of the file extension

File: index.html [hunk 5]
```diff
@@ -9,22 +9,17 @@
...
+        <li><a href="./Menu.html"> Menu </a></li>
+        <li><a href="./Reservations.htm ">Reservations</a></li>
+        <li><a href="./contactus.htm ">Connect with Us!</a></li>
```
Comment (billythakidd04) [inline index.html]: more extra spaces

**Review by billythakidd04 — CHANGES_REQUESTED** [review]
There are a bunch of changes that need to be made here. Also the filenames should all be the same php OR html. As I said in class last week `.htm` is outdated and `.html` is the correct form.

File: css/style.css (hunk 2 above)
Comment (rbaia28) [reply css/style.css]: yes, sorry, old sample

**Comment by billythakidd04** [comment]
Why was this closed?!

**Comment by rbaia28** [comment]
> Why was this closed?!

I wanted to resubmit a fresh pull request with all of the updates made for clarity for all team members

**Comment by billythakidd04** [comment]
> > Why was this closed?!
> 
> I wanted to resubmit a fresh pull request with all of the updates made for clarity for all team members

I don't understand... Is this pr missing something? I don't see any conflicts...

**Review by cwlainson — DISMISSED** [review]
good cleanup of white space and filename fixes
like that you've changed your pizza choice, no judgement on the pineapple

File: index.html [hunk 6]
```diff
@@ -9,22 +9,17 @@
...
+    <img src="img/PizzaParty.jpg" alt="Two slices of Pizza holding hands" width="300" height="400">
+    <img src="img/tmp.gif" alt="DJ CAT" width="300" height="400">
 
     <ul>
-        <a href="./index.html">Best Pizza in Town!</a>
-        <li><a href="./ourstory.htm">Our Story</a></li>
-        <li><a href="./Menu.htm"> Menu </a></li>
-        <li><a href="./Reservations.htm">Reservations</a></li>
-        <li><a href="./contactus.htm">Connect with Us!</a></li>
+        <a href=" ./index.html">Best Pizza in Town!</a>
```
Comment (billythakidd04) [inline index.html]: this path is still wrong

File: index.html [hunk 7]
```diff
@@ -9,22 +9,17 @@
...
+        <li><a href="./ourstory.html">Our Story</a></li>
+        <li><a href="./Menu.html"> Menu </a></li>
+        <li><a href="./Reservations.html">Reservations</a></li>
+        <li><a href="./contactus.html">Connect with Us!</a></li>
```
Comment (billythakidd04) [inline index.html]: actually all of these paths are wrong. there is no need for the `./`

**Review by billythakidd04 — CHANGES_REQUESTED** [review]

File: index.html (hunk 7 above)
Comment (cwlainson) [reply index.html]: hit "resolve conversation"  by accident

File: css/style.css [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+        text-align: left;
+    }
+    
+    .overlay img {
+        opacity: 0.5;
+    }
+    
+    title {
+        text-align: center;
+    }
```
Comment (billythakidd04) [inline css/style.css]: need a newline at the end of this file

File: index.html [hunk 9]
```diff
@@ -2,23 +2,35 @@
...
+            <h1>Pizza Party
+
+                <div class="topnav">
+                    <a href="./ourstory.html ">Our Story</a>
+                    <a href="./Menu.html ">Menu</a>
+                    <a href="./Reservations.html ">Reservations</a>
+                    <a href="./contactus.html ">Connect with Us!</a>
+                </div>
+            </h1>
+        </div>
```
Comment (billythakidd04) [inline index.html]: These should not be _inside_ the `<head>` tag

Comment (billythakidd04) [inline index.html]: These lines should not be _inside_ of `<head>` tag

File: index.html [hunk 10]
```diff
@@ -2,23 +2,35 @@
...
-
-    <ul>
-        <a href=" ./index.html">Best Pizza in Town!</a>
-        <li><a href="./ourstory.html">Our Story</a></li>
-        <li><a href="./Menu.html"> Menu </a></li>
-        <li><a href="./Reservations.html">Reservations</a></li>
-        <li><a href="./contactus.html">Connect with Us!</a></li>
-    </ul>
+    <img src=" img/PizzaParty.jpg " alt="Two slice of Pizza holding hands " width="300 " height="400 ">
+    <img src="img/pizza-3007395_960_720.jpg " alt="Pizza with toppings " width="500 " height="400 ">
```
Comment (billythakidd04) [inline index.html]: Move width and height to css

File: menu.html [hunk 11]
```diff
@@ -1,57 +1,86 @@
 <!DOCTYPE html>
 <html>
+
+<head>
+    <title>PizzaParty</title>
+    <link href="css/style.css" rel="stylesheet">
+</head>
 <meta charset="UTF-8">
 <title>Menu</title>
```
Comment (billythakidd04) [inline menu.html:9]: these lines _do_ need to be inside of `<head>` tag

File: menu.html [hunk 12]
```diff
@@ -1,57 +1,86 @@
...
-            <thead>
-                <th>Personal-12"</th>
-                <th>Large-18"</th>
-            </thead>
+            <tr>
+                <td>Cheese</td>
+                <td>$12.00</td>
+                <td>$18.00</td>
+            </tr>
+
```
Comment (billythakidd04) [inline menu.html]: no need for this extra line

File: menu.html [hunk 13]
```diff
@@ -1,57 +1,86 @@
...
+            <tr>
+                <td>Cheese with One Topping</<td>
+                    <td>$14.00</td>
+                    <td>$20.00</td>
+            </tr>
+
```
Comment (billythakidd04) [inline menu.html]: here too

File: menu.html [hunk 14]
```diff
@@ -1,57 +1,86 @@
...
 
-    <h1><b>Grandma's Pizza</b><span>$12.50</span></h1>
-    <P>Grandma pizza is a distinct pizza that originates from Long Island, New York. It is a thin, square pizza with Cheese, Tomatoes, and Fresh Basil</P>
-    <hr>
+        <hr>
 
-    <h1><b>Buffalo-Style</b></h1>
-    <p></p>
-    <hr>
+        <h1><b>Flyers</b><span>$16.50</span></h1>
```
Comment (billythakidd04) [inline menu.html]: what is this `<span>` tag for?

File: ourstory.html [hunk 15]
```diff
@@ -1,20 +1,9 @@
 <!DOCTYPE html>
+<html>
+
```
Comment (billythakidd04) [inline ourstory.html:3]: no `<head>` tag, no `<title>` tag, not even a comment... What is this page?

File: ourstory.html [hunk 16]
```diff
@@ -1,20 +1,9 @@
...
-        <img src="/Users/christopherlainson/Desktop/workspace/PizzaParty/img/party_boys.jpg" 
-        Alt="Pardi Bros Havana NYE 1958 colorized" style="width:560px;height:325px"><p>Carlo and Giovanni Pardi NYE 
-            Havana, Cuba 1958 (colorized).</p>
-        <iframe width="560" height="315" src="https://www.youtube.com/embed/CPEwAA6G42I?start=15" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+    <iframe width="560" height="315" src="https://www.youtube.com/embed/CPEwAA6G42I?start=15" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 
 </body>
-</html>
+
+</html>
```
Comment (billythakidd04) [inline ourstory.html]: missing newline at end of file

**Review by billythakidd04 — CHANGES_REQUESTED** [review]

**Comment by billythakidd04** [comment]
@gc824 there still is no description for this pr, please update that

File: index.html (hunk 6 above)
Comment (gc824) [reply index.html]: this is fixed

File: index.html (hunk 7 above)
Comment (gc824) [reply index.html]: this is fixed

File: css/style.css (hunk 8 above)
Comment (gc824) [reply css/style.css]: this is fixed

File: index.html (hunk 9 above)
Comment (gc824) [reply index.html]: this is fixed

Comment (gc824) [reply index.html]: this is fixed

File: index.html (hunk 10 above)
Comment (gc824) [reply index.html]: this is fixed

File: menu.html (hunk 11 above)
Comment (gc824) [reply menu.html:9]: this is fixed

File: menu.html (hunk 12 above)
Comment (gc824) [reply menu.html]: this is fixed

File: menu.html (hunk 13 above)
Comment (gc824) [reply menu.html]: this is fixed

File: menu.html (hunk 14 above)
Comment (gc824) [reply menu.html]: Removed span tags

File: ourstory.html (hunk 16 above)
Comment (gc824) [reply ourstory.html]: this is fixed

**Review by billythakidd04 — APPROVED** [review]

**Pull request merged**